docs/layouts/shortcodes/javadoc.html

```diff
@@ -16,11 +16,11 @@
 specific language governing permissions and limitations
 under the License.
 */}}{{/*
-    Shortcode for linking to Flinks JavaDoc
+    Shortcode for linking to Flink's JavaDoc
 
     Parmeters: 
         - name: The rendered link name (required)
 */}}
-<a href="{{ .Site.Params.JavaDoc }}/{{ .Get "file" }}">
-    {{ .Get "name" }}
+<a href="{{ .Site.Params.JavaDocs }}/{{ .Get "file" }}">
+    {{ .Get "name" | markdownify }}
```

Comment:
what is the goal here?

Reply:
> Could you open a JIRA ticket for this PR? I believe it is large enough to warrant one.

Sure. https://issues.apache.org/jira/browse/FLINK-21717

Reply:
> what is the goal here?

`JavaDoc` is a typo. The params defined in config.toml is `JavaDocs`.

Reply:
And what about `markdownify`? As in, what problems existed without it?

Reply:
`{{ page.excerpt | markdownify }}` is illustrated in https://jekyllrb.com/docs/liquid/filters/ that converts a Markdown-formatted string into HTML.  See also `docs/layouts/shortcodes/gh_link.html`.